### ISSUE TITLE
Error saving modified subtitles on Windows

### ISSUE THREAD

**Issue by carlitossuper1** [teras/Jubler#22]
As the title says, I can't save a modified subtitle on Windows.

When I double clic a subtitle file in windows explorer, it opens on Jubler, I do a modification, but when saving I get a message that the .srt file is unwritable. Obviously the file is not open by another app. I can also open Jubler from start menu, manualy open the sub, modify it and then I get the same error. I have to Save as.. to save as a new subtitle.

Can anyone test?

![image](https://user-images.githubusercontent.com/3442746/92997568-44116b80-f4e2-11ea-85a5-a46314bb3249.png)

**Comment by teras** [comment]
Maybe the file is indeed unwritable or locked by something else?
Permissions in Windows are a bit tricky.

**Comment by NostromoRetro** [comment]
Happens exactly the same to me. Sometimes I can save sometimes I cannot, and I have to choose "save as" (and duplicate the file) which is annoying

**Comment by carlitossuper1** [comment]
Exactly, usually happens for me when I play a video file with subtitles, AFTER I close the player and open the sub on Jubbler, then I can't save the edited subtitles.

Sometimes I can. It's really weird. 

I also can assure that even having the Media Player Classic open doesn't lock the .srt file because I can overwrite the file on the File Explorer or even redownloading the sub with the same name and it's replaced fine.

**Comment by teras** [comment]
Sometimes Windows insists on locking a file. Some (native, not Java) apps, under certain circumstances they can unlock a file if ACLs allow it. Even being locked for reading, under Windows is considered lock.
In other words, under Windows the whole lock mechanism is too complicated. And this is my own personal reason why I hate developing (and using in general) on Windows.
